src/main/resources/native/android/android_project/app/build.gradle

```diff
@@ -12,4 +12,13 @@ android {
         implementation 'com.android.support:support-v4:27.1.1'
         api fileTree(dir: '../libs', include: '*.jar')
     }
+
+    signingConfigs {
+        release {
+            storeFile file("KEYSTORE_FILE")
```

Comment:
I'm not fond of adding the passwords to the file. I'd rather have them in a gradle.properties file, that can be global or local to the project.
like:
```
STOREPASS=real_store_password
...
```
These properties can be also set by the client maven plugin, like:
```
<providedKeyStorePassword>STOREPASS</providedKeyStorePassword >
...
```

Reply:
Does this fix look ok to you?